code/modules/surgery/organs/tongue.dm

```diff
@@ -35,6 +36,8 @@
 		M.dna.species.say_mod = initial(M.dna.species.say_mod)
 
 /obj/item/organ/tongue/can_speak_in_language(datum/language/dt)
+	if(istype(src, /obj/item/organ/tongue/lizard) && istype(dt, /datum/language/human))
```

Comment:
lizards would still be able to speak human with a tongue transplant if they found a way to learn the language. you should do a species check on the owner instead